### PR TITLE
Fix border-inline-start and border-inline-end not having any valid ordering

### PR DESCRIPTION
**Pull request by dddlr**
### What is this change?

Fix how `border-inline-start` and `border-inline-end` does not have any valid ordering in the `shorthand-property-sorting` ESLint rule.

Follow up to https://github.com/atlassian-labs/compiled/pull/1749

### Why are we making this change?

Fix one more false positive that slipped through the cracks.

---

### PR checklist

Don't delete me!

I have...

- [x] Updated or added applicable tests
- [x] Updated the documentation in `website/`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -158,5 +158,6 @@
         "react"
       ]
     }
-  ]
+  ],
+  "packageManager": "yarn@1.22.21+sha256.dbed5b7e10c552ba0e1a545c948d5473bc6c5a28ce22a8fd27e493e3e5eb6370"
```
Comment (kylorhall-atlassian) [inline package.json:162]: What's this about?

Comment (dddlr) [reply package.json]: gahhh yarn keeps adding that

Comment (dddlr) [reply package.json]: we're supposed to specify what version of yarn the repo should use through `packageManager`

i never got the chance to set this up properly, so it keeps trying to add it whenever i run `yarn install`

maybe i should just set it up 🤔 hold on

Comment (dddlr) [reply package.json]: hmm looks a bit complicated, something for another day...

**Review by kylorhall-atlassian — APPROVED** [review]
Seems fair—worst-case it's an edge-case for us we might want to handle better _one day_, but for now it's perfectly acceptable.

File: package.json (hunk 1 above)
Comment (kylorhall-atlassian) [reply package.json]: Yeah no clue 🫡

**Pull request merged**